app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt

```diff
@@ -45,26 +45,30 @@ fun PlayerControls(
 
         Row(
             modifier = Modifier.align(Alignment.Center),
-            horizontalArrangement = Arrangement.spacedBy(2.dp),
+            horizontalArrangement = Arrangement.spacedBy(40.dp),
             verticalAlignment = Alignment.CenterVertically
         ) {
             IconButton(onClick = onSkipPrevious) {
                 Icon(
+                    modifier = Modifier.size(30.dp),
                     imageVector = Icons.Default.SkipPrevious,
                     contentDescription = stringResource(R.string.skip_previous)
                 )
             }
 
-            IconButton(onClick = onClickPlayPause) {
+            IconButton(
+                modifier = Modifier.
+                then(Modifier.size(64.dp)),
```

Comment:
No need for `.then`, you can just set the size directly

Reply:
okay, thank, i was copying stackoverflow cuz icon wasn't going above 40.dp on its own